_episodes/02-top-level-menu-features.md

```diff
@@ -1,36 +1,83 @@
 
-#### MarcEdit Tools: Features
-When you click on MARCEdit Tools Icon from the Main Menu, the available features are:
+>## How many shortcuts can be added to the main window?
+>
+> > ## Solution
+> > 1. Click on the gear icon in the main window or go to Tools -> Preferences
+> > 2. In the new window, make sure you select Main Window in the left hand pane
+> > 3. In the right pane, select any tool that hasn't been selected
+> > 5. You will see a popup message that says "Only four items can be select. Unselect on one of the options and then reselect."
+> > 6. A user can only select 4 options or tools as shortcuts to appear in the main window.
+> {: .solution}
+{: .challenge}
+
+If you change your preferences for the MarcEditor, the tool used to work with MARC data, you can always go back to the default settings. 
+>## Reset settings for the MarcEditor in Preferences
+>1. Click on the gearbox in the top left corner or go to Tools -> Preferences
+>2. Make sure you select "MarcEditor" in the Preferences window in the left hand pane
+>3. In the right hand pane, select Set Defaults for either font or font size
+>3. Click Ok
+{: .checklist}
+
+>## Name three other tools or features that can be changed in Preferences.
+>
+> > ## Solution
+> > 1. Click on the gear icon in the main window or go to Tools -> Preferences
+> > 2. In the left hand pane, look at the names there
+> > 3. Tools or features that can be changed in Preferences: Main Window, Bibframe, MarcEditor, MARCEngine, Ease of Use, File Associations, Language, Locations, Other, Updates, Configure Watcher, ILS Integration, OCLC API Integration.
+> {: .solution}
+{: .challenge}
+
+### MarcEdit Main Window: Top Level Menu
+In the main window top level menu, there are three options: Tools, Plug-ins, and Help. 
+
+#### Plug-Ins
+There are a number of useful plugins that can be added to MarcEdit to extend its functionality. An example is the MARC to [KBART or Knowledge bases and related tools](https://www.niso.org/standards-committees/kbart/kbart-frequently-asked-questions) converter. You can find more information on plugins on the [MarcEdit Plugin page](https://marcedit.reeset.net/managing-plugins-in-marcedit).
+
+#### Help
+This top level menu Help allows you to find information on updates, about your current MarcEdit installation, and links out to video tutorials or online help for instance. 
+
+#### Tools
+The Tools menu allows you to access a number of different features such as Prefernces which we saw above. Other features to highlight are:
+* The MarcEdit regular expression store
+* OAI Harvester
+* Export either MARC records or records reformatted as a text delimited csv (comma separated value) or tsv (tab separated value) file
+* MARC Processing Tools: MARCsplit, MARCcompare, MARCjoin, MARCmerge, find duplicate records, RDA Helper, or MARCValidator for instance
+* Delimited Text Translator
+* Preferences
+* Utilities that include a verify URL tool
+
+#### MARCTools or the tools icon
```

Comment:
Would it make sense for preferences to come after tools so that students know what tools are before they select/update defaults?

Reply:
I noticed that preferences came up before tools which might not make sense for the flow either.